<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsFrameView</class>
 <widget class="QDockWidget" name="SettingsFrameView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>373</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Frame View Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="iconWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <item>
         <property name="text">
          <string>Find</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/find.ico</normaloff>:/images/find.ico</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Filters</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/filter.ico</normaloff>:/images/filter.ico</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tools</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/tools.ico</normaloff>:/images/tools.ico</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/view.ico</normaloff>:/images/view.ico</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="findPage"/>
        <widget class="QWidget" name="filtersPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolsPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QPushButton" name="applyButton">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="fitAllButton">
              <property name="text">
               <string>Fit All</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>View :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="fitSelectionButton">
              <property name="text">
               <string>Fit Selection</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="resetButton">
              <property name="text">
               <string>Reset View</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="saveButton">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="showCheckBox">
              <property name="text">
               <string>show</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="clearDataButton">
              <property name="text">
               <string>Clear Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="Color">
             <attribute name="title">
              <string>Color</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="beginSpinBox">
                  <property name="maximum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="endSpinBox">
                  <property name="maximum">
                   <double>100.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QCheckBox" name="color4CheckBox">
                  <property name="text">
                   <string>Color 4</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QDoubleSpinBox" name="color1SpinBox">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QCheckBox" name="color1CheckBox">
                  <property name="text">
                   <string>Color 1</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QtColorPicker" name="color1Button">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QDoubleSpinBox" name="color3SpinBox">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="8" column="2">
                 <widget class="QtColorPicker" name="color4Button">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QCheckBox" name="color2CheckBox">
                  <property name="text">
                   <string>Color 2</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Interval End</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="2">
                 <widget class="QtColorPicker" name="color2Button">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QDoubleSpinBox" name="color2SpinBox">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Interval Begin</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="3">
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="7" column="2">
                 <widget class="QtColorPicker" name="color3Button">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QDoubleSpinBox" name="color4SpinBox">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QCheckBox" name="color3CheckBox">
                  <property name="text">
                   <string>Color 3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="viewPage"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
	 <header location="global">3rd/qtsln/qtcolorpicker/qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>color1CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>color1SpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color1CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>color1Button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color2CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>color2SpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color2CheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>color2Button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
